polyfills/HTMLPictureElement/config.json

```diff
@@ -11,9 +11,11 @@
 		"bb": "7 - 10",
 		"android": "* < 46"
 	},
-	"dependencies": [],
+	"dependencies": ["html5shiv"],
 	"license": "MIT",
 	"spec": "https://html.spec.whatwg.org/multipage/embedded-content.html#embedded-content",
 	"repo": "https://github.com/scottjehl/picturefill",
-	"docs": "https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture"
+	"docs": "https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture",
+	"module":"picturefill",
```

Comment:
I think `module` and `paths` should be grouped under something like `install` or `update`, like we do with `build`.